### DISCUSSION TITLE
Does argo rollout support header based traffic control in AWS App Mesh

### DISCUSSION THREAD

**Discussion by g-traveller** [argoproj/argo-rollouts#2825] [General]
Hi, Team,

Our service deployed on AWS EKS with AppMesh, ArgoCD is our CD tool, we need header based traffic management.
Is possible Argo Rollout support header based traffic control in AWS App Mesh

Thanks

**Comment by g-traveller** [comment]
any update?

**Reply by zachaller** [reply]
It does not but would be open to a feature contribution
